packages/flutter/test/semantics/semantics_test.dart

```diff
@@ -337,6 +337,7 @@ void main() {
       '   mergeAllDescendantsIntoThisNode: false\n'
       '   Rect.fromLTRB(0.0, 0.0, 0.0, 0.0)\n'
       '   actions: []\n'
+      '   customActions: []\n'
```

Comment:
Maybe add a test that ensures that custom actions are properly printed when the Semantics tree is printed out?

Reply:
Done